_studio/hevc_fei/CMakeLists.txt

```diff
@@ -21,72 +21,45 @@
 if(API_VERSION VERSION_LESS "1.27")
   message("Skipping HEVC FEI plugin due to API restriction")
 else()
+  set(MFX_ORIG_LDFLAGS "${MFX_LDFLAGS}" )
 
-set(MFX_ORIG_LDFLAGS "${MFX_LDFLAGS}" )
+  mfx_include_dirs( )
 
-set(HEVC_FEI_Encoder_HW_GUID "5418a70666f94d5cb4f7b1caee86339b")
+  set(HEVC_FEI_Encoder_HW_GUID "5418a70666f94d5cb4f7b1caee86339b")
 
-# Plugin version info
-set (HEVC_FEI_HW_PRODUCT_NAME "Intel(R) Media Server Studio 2018 - HEVC FEI Encode")
-set (HEVC_FEI_HW_DESCRIPTION "Intel(R) Media Server Studio 2018 - HEVC FEI Hardware Encode Plug-in")
```

Comment:
Please leave wording "Hardware Encode Plug-in". Previously absence of "Hardware" meant software component.